presto-spi/src/main/java/com/facebook/presto/spi/StandardErrorCode.java

```diff
@@ -132,6 +132,7 @@ public enum StandardErrorCode
     NATIVE_EXECUTION_PROCESS_LAUNCH_ERROR(0x0002_000F, INTERNAL_ERROR),
     MISSING_RESOURCE_GROUP_SELECTOR(0x0002_0010, INTERNAL_ERROR),
     EXCEEDED_HEAP_MEMORY_LIMIT(0x0002_0011, INSUFFICIENT_RESOURCES),
+    EXCEEDED_WRITTEN_INTERMEDIATE_BYTES_LIMIT(0x00020012, INSUFFICIENT_RESOURCES),
```

Comment:
should this be 0x0002_0012?

Reply:
I think the _ is just to make it readable but sure i can update

Reply:
Let's make this update to keep it consistent.